packages/now-static-build/test/fixtures/45-node-modules-bin-path/now.json

```diff
@@ -12,7 +12,8 @@
   ],
   "build": {
     "env": {
-      "NOW_BUILDER_DEBUG": "1"
+      "NOW_BUILDER_DEBUG": "1",
+      "VERCEL_BUILDER_DEBUG": "1"
```

Comment:
Why are both of these necessary?

Reply:
I think we'll have to deploy `@vercel/build-utils` for this change to take affect on `@vercel/static-build` here but maybe my assumption was wrong. Let me try it.